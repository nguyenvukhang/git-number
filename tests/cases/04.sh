# unstaged modified file (filename)

init 2
echo "_" >gold
_gitnu add gold # use filename
_gitnu commit -m "first"
echo "_" >>gold
save gitnu status
