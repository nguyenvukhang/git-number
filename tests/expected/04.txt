On branch main
Changes not staged for commit:
1	[31mmodified:   gold[m

Untracked files:
2	[31mfile_0001[m
3	[31mfile_0002[m

no changes added to commit
